session/txn.go

```diff
@@ -461,7 +461,9 @@ func (s *session) StmtCommit() error {
 // StmtRollback implements the sessionctx.Context interface.
 func (s *session) StmtRollback() {
```

Comment:
@tiancaiamao PTAL  if we can change `StmtRollback` like this

Reply:
That's ok.
B.T.W, I'm thinking of removing the assertion proto which is introduced to add an additional check for write consistency, but I find it's hard to achieve that goal.